### PR TITLE
Windows: Rename VSINSTALLDIR to LDC_VSDIR

### PR DESCRIPTION
**Pull request by kinke**

### CODE REVIEW

File: windows/build_release.bat [hunk 1]
```diff
@@ -1,13 +0,0 @@
-setlocal
```
Comment (rainers) [inline windows/build_release.bat:1]: I usually use this batch for local testing. Is there a way to execute (part of) the azure-scripts instead?

**Review by rainers — APPROVED** [review]
Otherwise LGTM

File: windows/build_release.bat (hunk 1 above)
Comment (AndrewEdwards) [reply windows/build_release.bat:1]: I'm not sure about that. Let me defer to the professionals. @kinke, @thewilsonator, @CyberShadow. Can you provide a response to this question? Thanks.

Comment (kinke) [reply windows/build_release.bat:1]: > Is there a way to execute (part of) the azure-scripts instead?

Not that I know of.

I found it better to have all logic in the YAML, where the directories for PATH are set up appropriately (e.g., I had no idea that the `dm` dir is supposed to be the DMC dir augmented by implib.exe).

**Comment by kinke** [comment]
Pinging @MartinNowak, as the Vagrant/VM setup will most likely have to be adapted.

**Comment by MartinNowak** [comment]
Done https://gist.github.com/MartinNowak/8270666/revisions#diff-de5ae8181f71cdfb109e9e8d7fa69883

**Pull request merged**